packages/nimble/R/genCpp_generateCpp.R

```diff
@@ -187,7 +202,7 @@ cppOutputFor <- function(code, symTab) {
     begin <- nimGenerateCpp(code$args[[2]]$args[[1]], symTab)
     end <- nimGenerateCpp(code$args[[2]]$args[[2]], symTab)
     iterVar <- nimGenerateCpp(code$args[[1]], symTab)
-    part1 <- paste0('for(', iterVar ,'=', begin,'; ', iterVar, '<= static_cast<int>(', end, '); ++', iterVar,')')
+    part1 <- paste0('for(', iterVar ,'=', begin,'; ', iterVar, '<= static_cast<int>(', end, '); ', iterVar,'++ )')
```

Comment:
Yes that change makes sense.  I think I've read some claims that a compiler can make them equivalent if possible, but I could be mis-remembering or it may be incorrect.